### PR TITLE
Distillation qanda

### PR DESCRIPTION
**Pull request by visahan-24**
# Description

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_qanda.ipynb [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+    "For our example, we download and use the Common Sense Q&A dataset (https://huggingface.co/datasets/tau/commonsense_qa/) from HuggingFace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datasets import load_dataset\n",
```
Comment (skasturi) [inline sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_qanda.ipynb:220]: Could we please move these to utils files?

Comment (visahan-24) [reply sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_qanda.ipynb]: @skasturi - Its updated

File: sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_nlu_qa_task.ipynb [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pick a teacher model\n",
+    "\n",
+    "We support **Meta-Llama-3.1-405B-Instruct** as the teacher model. \n",
+    "### First deploy the teacher model in Azure AI Studio\n",
+    "* Go to Azure AI Studio (ai.azure.com)\n",
```
Comment (balapv) [inline sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_nlu_qa_task.ipynb:151]:
```suggestion
    "* Deploy Meta Llama 3.1 405B Instruct as a serverless API - [link](https://aka.ms/meta-llama-3.1-405B-instruct-azure-ai-studio-docs#create-a-new-deployment)",
```

File: sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_nlu_qa_task.ipynb [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+    "* Select Meta-Llama-3.1-405B-Instruct model from Model catalog.\n",
```
Comment (balapv) [inline sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_nlu_qa_task.ipynb:152]:
```suggestion

```

File: sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_nlu_qa_task.ipynb [hunk 4]
```diff
@@ -0,0 +1,558 @@
...
+    "* Deploy with \"Pay-as-you-go\"\n",
```
Comment (balapv) [inline sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_nlu_qa_task.ipynb:153]:
```suggestion

```

File: sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_chat_completion.ipynb [hunk 5]
```diff
@@ -486,6 +486,7 @@
     "        learning_rate=0.00002,\n",
     "        per_device_train_batch_size=1,\n",
     "        num_train_epochs=3,\n",
+    "        data_generation_task_type=\"NLI\",\n",
```
Comment (skasturi) [inline sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_chat_completion.ipynb:489]: Could we please make this parameter optional, if not already?

Comment (visahan-24) [reply sdk/python/foundation-models/system/finetune/Llama-notebooks/distillation/distillation_chat_completion.ipynb:489]: @skasturi - If the parameter is optional, then the default data generation task is ambiguous. We have had a discussion about this and then it's marked as a mandatory parameter.

**Review by skasturi — APPROVED** [review]

**Review by scravikiran — APPROVED** [review]
LGTM

**Pull request merged**